docs/05_components/doc-management/doc-management-server/doc-mgmt-pipelines.mdx

```diff
@@ -0,0 +1,76 @@
+---
+title: 'Document Management - `Data Pipelines'
+sidebar_label: 'Data Pipelines'
+sidebar_position: 4
+id: doc-management-pipelines
+keywords: [doc-management, pipelines]
+tags:
+  - doc-management
+  - file-storage-pipelines
+---
+
+Here are some implementations provided with the Document Manager that can be used within any data pipeline you have set up. See [Data Pipelines](/server/data-pipelines/introduction/) for more information.
```

Comment:
We need to call out that you have to add a dependency on file-server-pipelines to access the gpal extension functions / classes